src/GitHub.Exports.Reactive/Models/IPullRequestSessionFile.cs

```diff
@@ -41,5 +41,11 @@ public interface IPullRequestSessionFile : INotifyPropertyChanged
         /// Gets the inline comments threads for the file.
         /// </summary>
         IReadOnlyList<IInlineCommentThreadModel> InlineCommentThreads { get; }
+
+        /// <summary>
+        /// Gets an observable that is raised with a collection of 0-based line numbers when the
```

Comment:
What do the line numbers represent? Are they where new comments have been added or deleted?

Reply:
They're the lines that have have a changed inline comment state, so either a comment was added or removed on that line. Feel free to suggest a better wording!

Reply:
That makes sense. :thumbsup: